numpy/testing/_private/utils.py

```diff
@@ -1768,12 +1768,10 @@ def assert_warns(warning_class, *args, **kwargs):
     ----------
     warning_class : class
         The class defining the warning that `func` is expected to throw.
-    func : callable
-        The callable to test.
-    \\*args : Arguments
-        Arguments passed to `func`.
+    \\*args : List of function and arguments
+        `func` and arguments for `func`.
```

Comment:
Apologies - this is correct, but I think these were clearer as separate entries.

Reply:
@eric-wieser Would you like me to revert this?

Reply:
The rewording is fine, but I think it would be better to list the arguments separately.